src/parser/hevc_parser.cpp

```diff
@@ -1542,6 +1543,7 @@ ParserResult HevcVideoParser::ParseSliceHeader(uint8_t *nalu, size_t size, HevcS
 
     // Set active VPS, SPS and PPS for the current slice
     m_active_pps_id_ = Parser::ExpGolomb::ReadUe(nalu, offset);
+    CHECK_ALLOWED_MAX(m_active_pps_id_, (MAX_SPS_COUNT - 1));
```

Comment:
Should check against MAX_PPS_COUNT, not MAX_SPS_COUNT.

Reply:
I thought sps_id ranges from 0 - (count-1)